### PR TITLE
chore(echo): rename @novu/echo to @novu/framework

### PR DESCRIPTION
**Pull request by denis-kralj-novu**
### What changed? Why was the change needed?
- Update package json
- Update package folder name
- Update readme details

### CODE REVIEW

File: packages/framework/README.md
Comment (denis-kralj-novu) [inline packages/framework/README.md:1]: First place with actual changes, the consideration is that we are updating the docs URL while we still don't have them updated, should this be included now or after we update the docs?

File: packages/framework/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@novu/echo",
-  "version": "0.24.3-alpha.4",
+  "name": "@novu/framework",
+  "version": "0.24.3-alpha.5",
```
Comment (denis-kralj-novu) [inline packages/framework/package.json:3]: updated alpha version here

Comment (denis-kralj-novu) [reply packages/framework/package.json:3]: Also, the author is set as "Athena", we wan't to change that to "Novu" maybe?

Comment (rifont) [reply packages/framework/package.json:3]: Yes, let's make it `Novu`

**Review by rifont — APPROVED** [review]
🪟🏗️ 🚀

**Pull request merged**